ocs_ci/deployment/mce.py

```
@@ -646,11 +646,20 @@ def wait_csv_upgraded(self):
-            kind=constants.CLUSTER_SERVICE_VERSION,
-            namespace=self.mce_namespace,
+        sampler = TimeoutSampler(
+            timeout=300,
+            sleep=10,
+            func=self.get_mce_csv_name,
         )
-        csv_name = csv_ocp_obj.get(resource_name="")["items"][0]["metadata"]["name"]
+        csv_name = ""
+        for csv_name in sampler:
+            if csv_name:
+                logger.info(f"Found MCE CSV: {csv_name}")
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)